tests/forte/data/readers/sst2_reader_test.py

```diff
@@ -112,8 +112,7 @@ def test_reader(self):
                 self.assertEqual(leaf_node_4.text, "biopic")
                 self.assertEqual(leaf_node_4.is_leaf, True)
                 self.assertEqual(leaf_node_4.parent_node, right_subtree)
-            print(i)
-        self.assertEqual(count_pack, 11855)
+        self.assertEqual(count_pack, 1)
```

Comment:
By the way, I also realized you are putting one sentence per pack. It is OK but this may include too many packs if we write them to disk. Maybe we can put a configurable number here to include, say 100 sentences, per pack?